pensjonsmaler/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/alderspensjon/AlderspensjonFelles.kt

```diff
@@ -141,9 +147,9 @@ data class ArbeidsinntektOgAlderspensjon(
         showIf(uttaksgrad.equalTo(100)) {
             paragraph {
                 text(
-                    Bokmal to "Hvis du har 100 prosent alderspensjon, gjelder økningen fra 1. januar året etter at skatteoppgjøret ditt er ferdig.",
-                    Nynorsk to "Dersom du har 100 prosent alderspensjon, gjeld auken frå 1. januar året etter at skatteoppgjeret ditt er ferdig.",
-                    English to "If you are receiving a full (100 percent) retirement pension, the increase will come into effect from 1 January the year after your final tax settlement has been completed.",
+                    Bokmal to "Hvis du har 100 prosent alderspensjon, gjelder økningen fra 1. januar året etter at skatteoppgjøret ditt er ferdig.",
+                    Nynorsk to "Dersom du har 100 prosent alderspensjon, gjeld auken frå 1. januar året etter at skatteoppgjeret ditt er ferdig.",
+                    English to "If you are receiving a full (100 percent) retirement pension, the increase will come into effect from 1 January the year after your final tax settlement has been completed.",
                 )
             }
         }.orShow {
```

Comment:
.orShowIf(not(uforeKombinertMedAlder)), trur eg

Reply:
Eller kanskje ikkje? Fekk det inntrykket frå 92-brevet, men ikkje frå 97-brevet

Reply:
Har tilført mer logikk